_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,254 @@
+  [proposed][Bitcoin Core #19204] that this could be done instead using
+  the [BIP133][] `feefilter` message.  That eliminates any need for a
+  non-bloom node to support the `filterclear` message and so this PR
+  removes that feature.
+
+- [Bitcoin Core #19133][] adds a bitcoin-cli `-generate` parameter (note leading dash) to replace
+  the functionality of the bitcoind `generate` RPC method, which was removed
+  from Bitcoin Core in [Version 0.19.0.1][core version]. The new
+  implementation avoids creating unnecessary dependencies between the
+  wallet and other components. Providing an alias for the old RPC is useful for
+  testing and makes it easier to update [broken documentation][broken docs].
+
+- [Bitcoin Core #18027][] adds two options to the GUI's *File* menu for
+  working with Partially Signed Bitcoin Transactions ([PSBTs][topic
+  psbt]): *Load PSBT from file* and *Load PSBT from clipboard*.  When
+  one of those options is clicked and a PSBT is loaded, a dialog is
+  provided that allows the user to sign an incomplete PSBT if this
+  wallet has its key, broadcast a complete PSBT, or copy or save the
+  PSBT for processing with another tool such as [HWI][].  Combined with
+  other recent PSBT-related GUI changes (see Newsletters [#74][news74
+  psbtgui] and [#82][news82 psbtgui]) and [HWI's own GUI][hwi gui], this
+  makes it possible to use a PSBT-based process with Bitcoin Core for
+  the first time without using any RPCs.
+
+    ![Screenshot of PSBT dialog in Bitcoin Core GUI](/img/posts/2020-06-psbt-dialog.png)
+
+- [Bitcoin Core #16377][] updates the `walletcreatefundedpsbt` and
+  `fundrawtransaction` RPCs.  These RPCs normally use the wallet to
+  automatically choose which UTXOs to spend in an unsigned transaction,
+  but they also allow the user to specify one or more UTXOs they want to
+  spend in that transaction.  Previously, if the UTXOs selected by the
+  user weren't enough to pay for all of the transaction's outputs, the
+  wallet would automatically select more UTXOs to spend.  But, if a
+  user is manually selecting UTXOs, they may have some reason they don't
+  want to spend additional UTXOs, so the RPCs will now fail by default
+  if the user manually selects any UTXOs.  This default may be overridden
+  using the new `add_inputs` parameter to both RPCs.
+
+- [Eclair #1461][] adds several API endpoints that forward to Bitcoin
+  Core RPCs for relaying that program's wallet balances and other
+  information.  The goal is to make it easier to integrate Eclair with
+  the [Ride The Lightning][] node management dashboard.
+
+- [Bitcoin Core #19071][] adds documentation describing how developers
+  can contribute to the new and experimental [Bitcoin Core GUI
+  repository][].  Pull requests related to the GUI should be made into
+  this new repository, which will be bidirectionally synced with the
+  [main repository's master development branch][bitcoin/bitcoin] using
```

Comment:
can omit "development", or as an alternative (given possible adoption): "primary repository's main branch" (as per https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references)